components/AppMetadata/AppMetadata.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./AppMetadata.module.css";
+
+type Props = {
+  minSaleorVersion: string;
+  roadmapUrl: string;
+  githubUrl: string;
+};
+
+export const AppMetadata = ({
+  minSaleorVersion,
+  roadmapUrl,
+  githubUrl,
+}: Props) => {
+  return (
+    <div className={styles.root}>
+      <div>
+        <span>
+          Saleor version required: <strong>{minSaleorVersion}</strong>
+        </span>
+      </div>
+      <div>
+        <span>
+          Roadmap: <a href={roadmapUrl}>GitHub</a>
+        </span>
+      </div>
+      <div>
+        <span>
+          Repository: <a href={githubUrl}>GitHub</a>
```

Comment:
**suggestion (non-blocking)**: add `target="_blank"`

Reply:
**suggestion (non-blocking)**: IMO, it's more aesthetic. 

```suggestion
          <a href={githubUrl}>Repository</a>
```

Reply:
I'd wait for a design of the component